### ISSUE TITLE
separate training and run/test model classes

### ISSUE THREAD

**Issue by taimoorkhan-nlp** [Alaaeddin103/Rumor_verification#4]
Further, i would suggest splitting rumor classifier to two. i) training rumor classifier model, where we have labels, evidence and will be finetuning the model ii) test/run rumor classifier model, were we will provide the test data to the model to make predictions. 

Evaluations will be applied on the later.

**Comment by Alaaeddin103** [comment]
I added the run_test module, which receives relevant timeline entries from the retriever module and makes predictions using the fine-tuned model. It then provides the final stance based on the retrieved evidence, following the workflow you shared with me and added training modules two .
